backend/shared/rest/src/main/java/com/meemas/shared/Pair.java

```diff
@@ -0,0 +1,27 @@
+package com.meemas.shared;
+
+import lombok.Data;
+
+@Data
+public class Pair<K, V> {
+
+  private final K element0;
+  private final V element1;
+
+  public static <K, V> Pair<K, V> createPair(K element0, V element1) {
```

Comment:
```suggestion
  public static <K, V> Pair<K, V> create(K element0, V element1) {
```

"pair" bit in the method is redundant as it is obvious we are creating pair from usage, e.g. `Pair.create(...)`